src/main/java/uk/gov/companieshouse/api/accounts/model/validation/Error.java

```diff
@@ -0,0 +1,133 @@
+package uk.gov.companieshouse.api.accounts.model.validation;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.springframework.data.mongodb.core.mapping.Field;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Represents a validation error
+ */
+public class Error {
+
+    @JsonProperty("error")
+    @Field("error")
+    private String error;
```

Comment:
Sonar has raised an issue not sure how we tackle this given its copied from the spec:
> A field should not duplicate the name of its containing class

Reply:
@jsimons1989 Should we be changing the specs to stop Sonar issues?

Reply:
This will remain in place pending a further discussion with the architects it can be addressed as part of another change if needed.

Reply:
@leahtaylor @sroberts2 The class can be rename to ValidationError to avoid the issue in Sonar.